config/environment.rb

```diff
@@ -1,7 +1,19 @@
 # frozen_string_literal: true
 
+LOG_LEVELS = ['debug', 'info', 'warn', 'error', 'fatal', 'unknown']
```

Comment:
Can you not programmatically get a list of valid log levels from Rails, so as to avoid making this list yourself?

Reply:
couldn't get it. i don't think there's a risk here. for this to break we need that rails will add a severity _and_ that we will write logs in that severity. not likely to happen in any way.

Reply:
```ruby
2.3.8 :004 > Logger::Severity::constants
 => [:DEBUG, :INFO, :WARN, :ERROR, :FATAL, :UNKNOWN]
```
It's from plain Ruby and not Rails.